storybook/main.js

```diff
@@ -60,11 +60,6 @@ module.exports = {
 		name: '@storybook/react-webpack5',
 		options: {},
 	},
-	features: {
-		babelModeV7: true,
-		emotionAlias: false,
```

Comment:
We're using Emotion 11 so this is not necessary, and the flag was removed anyway, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#70-feature-flags-removed